supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-single-sign-on.adoc

```diff
@@ -0,0 +1,146 @@
+[[red-hat-single-sign-on-conventions]]
+
+[discrete]
+[[access-token]]
+== Access token
+A token that can be provided as part of an HTTP request that grants access to the service being invoked on. This is part of the OpenID Connect and OAuth 2.0 specification.
+
+[discrete]
+[[assertion]]
+== Assertion
+Information about a user. This usually pertains to an XML blob that is included in a SAML authentication response that provided identity metadata about an authenticated user.
+
+[discrete]
+[[authentication]]
+== Authentication
+The process of identifying and validating a user.
+
+[discrete]
+[[authentication-flows]]
+== Authentication flows
+Authentication flows are work flows a user must perform when interacting with certain aspects of the system. A login flow can define what credential types are required. A registration flow defines what profile information a user must enter and whether something like reCAPTCHA must be used to filter out bots. Credential reset flow defines what actions a user must do before they can reset their password.
+
+[discrete]
+[[authorization]]
+== Authorization
+The process of granting access to a user.
+
+[discrete]
+[[client-adapters]]
+== Client adapters
+Client adapters are plugins that you install into your application environment to be able to communicate and be secured by Red Hat Single Sign-On. Red Hat Single Sign-On has a number of adapters for different platforms that you can download. There are also third-party adapters you can use for environments that Red Hat do not cover.
```

Comment:
```Client adapters are plugins that you install into your application environment to be able to communicate and be secured by Red Hat Single Sign-On.```

I don't understand:

- who or what will be able to communicate after installing the plugins.
- what is being secured by SSO.

Reply:
@ndeevy Can you change this to 
Client adapters are libraries that make it easy to secure applications and services with Red Hat Single Sign-On.
Thanks, @stoobie